### PR TITLE
Add better-deal extension

### PR DESCRIPTION
**Pull request by juxtdesigncc**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/better-deal/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          "name": "unitSize",
+          "type": "text",
+          "placeholder": "Unit Size",
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "view",
+      "title": "View Better Deal",
```
Comment (pernielsentikaer) [inline extensions/better-deal/package.json:42]:
```suggestion
      "title": "View Better Deal History",
```
I would also add a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) that tells what the command is for, I was confused at first

File: extensions/better-deal/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "author": "juxtdesigncc",
+  "categories": [
+    "Productivity",
+    "Finance"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Add Better Deal",
```
Comment (pernielsentikaer) [inline extensions/better-deal/package.json:16]:
```suggestion
      "title": "Calculate Better Deal",
```
Maybe a beter name?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- I'm not sure what this extension does or how it can be helpful. Maybe if you add something in the README explaining it, it might make sense

-  Maybe you could add options to delete individual entries or all entries in the history?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Is this ready for another review?

**Comment by juxtdesigncc** [comment]
> Is this ready for another review?

Hi @pernielsentikaer! I'm happy to let you know that it is ready for another review. 😊

### Updated README for Extension Intro ([Link](https://github.com/raycast/extensions/pull/14180/commits/2ff1cdb10104b33cb5767093a96bb28e9567c5bb#diff-15fed497658e42ea736a18db8ac93fd2e25b90f2bde043284d0c18584f54850b))
> **Better Deal** is a handy calculator for online shopping that helps you find the better deal between two options by considering the price, quantity, and unit size. It also keeps track of your five most recent records for easy reference. For example, if you're shopping online and see two options for Coke—a 32-pack of 355ml cans and a 16-pack of 1L bottles—you can quickly input the prices and quantities of both options. The app will instantly calculate the cost per milliliter, helping you determine which pack offers the best value for your money. No more guesswork—just straightforward comparisons to ensure you get the most bang for your buck!

### Added Capabilities to Delete Individual Entries or All Entries ([Link](https://github.com/raycast/extensions/pull/14180/commits/da63d43a19dbe8e871ed4b58951215bb0b983354#diff-fce72e59c2e0e5052fb879bb234177fad01c80162d9b25cbdf73615928f74268))

### Changed Display of Each Entry to Show Record Updated Date & Time ([Link](https://github.com/raycast/extensions/pull/14180/commits/2ff1cdb10104b33cb5767093a96bb28e9567c5bb#diff-fce72e59c2e0e5052fb879bb234177fad01c80162d9b25cbdf73615928f74268))

Thank you for your support, and I look forward to your feedback!

**Comment by pernielsentikaer** [comment]
For actions deleting items we might want to add `style={Action.Style.Destructive}` - for the one deleting all items we might need to add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) with `      rememberChoice: true,` 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**